env/env_test.go

```diff
@@ -50,6 +49,26 @@ func TestPatchAll(t *testing.T) {
 	assert.DeepEqual(t, sorted(oldEnv), sorted(os.Environ()))
 }
 
+func TestPatchAllWindows(t *testing.T) {
+	skip.If(t, runtime.GOOS != "windows")
```

Comment:
Skiping on non-windows system as.. it is not allowed to set an environment variable starting with `=` on unix(es) 😓

Reply:
Do we have to skip this on Linux? The code itself doesn't exclude Linux (although `=` is invalid on linux in a name)

Reply:
Tested below with `TestToMap` 😉